Dockerfile

```diff
@@ -435,6 +441,9 @@ RUN ln -sf ${_CUDA_COMPAT_PATH}/lib.real ${_CUDA_COMPAT_PATH}/lib \
  && ldconfig \
  && rm -f ${_CUDA_COMPAT_PATH}/lib
 
+# Need to include pytorch in LD_LIBRARY_PATH since Torchvision loads custom ops
+ENV LD_LIBRARY_PATH /opt/tritonserver/lib/pytorch/:$LD_LIBRARY_PATH
```

Comment:
Move this up right after "ENV PATH" since they are related envvars. You don't need to restart CI.

"... since Torchvision loads custom ops from that path"